Sources/Datadog/Core/System/MobileDevice.swift

```diff
@@ -68,6 +73,24 @@ internal class MobileDevice {
                 )
             }
         )
+        
+        isLowPowerModeEnabled = processInfo.isLowPowerModeEnabled        
```

Comment:
Is `isLowPowerModeEnabled` meant to be used at line 72? It seems it is updated here and in the NotificationCenter block but never read.

P.S. stumbled upon this PR while casually browsing, just a quick comment.

Reply:
🤦